### ISSUE TITLE
NegativeArraySizeException

### ISSUE THREAD

**Issue by Encritary** [jython234/BouncyBall#11]
Hello, jython234! I have a problem: on trying decoding LoginPacket, BlockServerIOLib class - BinaryReader throws NegativeArrayException in function readString. I've update Login Packet, temporary delete logging of chat, and update protocol. BlockServerIOLib is outdated, yep?

**Comment by alejzeis** [comment]
Yes BlockServerIOLib is outdated. You can either ask @PEMapModder to help
update it (he made it) or you can look at BinaryBuffer in the main
BlockServer repository.

**Comment by Encritary** [comment]
Ok, I replaced it with Binary and BinaryStream from Nukkit
